src/NzbDrone.Core/Tv/SeriesEditedService.cs

```diff
@@ -1,24 +1,23 @@
-﻿using NzbDrone.Core.Messaging.Commands;
+﻿using NzbDrone.Core.MediaFiles;
 using NzbDrone.Core.Messaging.Events;
-using NzbDrone.Core.Tv.Commands;
 using NzbDrone.Core.Tv.Events;
 
 namespace NzbDrone.Core.Tv
 {
     public class SeriesEditedService : IHandle<SeriesEditedEvent>
     {
-        private readonly IManageCommandQueue _commandQueueManager;
+        private readonly IDiskScanService _diskScanService;
 
-        public SeriesEditedService(IManageCommandQueue commandQueueManager)
+        public SeriesEditedService(IDiskScanService diskScanService)
         {
-            _commandQueueManager = commandQueueManager;
+            _diskScanService = diskScanService;
         }
 
         public void Handle(SeriesEditedEvent message)
         {
             if (message.Series.SeriesType != message.OldSeries.SeriesType)
             {
-                _commandQueueManager.Push(new RefreshSeriesCommand(message.Series.Id));
+                _diskScanService.Scan(message.Series);
```

Comment:
Euh, no... if series type changes then a RefreshSeries is mandatory since the refresh logic has series type specific behavior.